bookkeeper-server/pom.xml

```diff
@@ -39,6 +39,11 @@
       <artifactId>bookkeeper-proto</artifactId>
       <version>${project.parent.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.rocksdb</groupId>
+      <artifactId>rocksdbjni</artifactId>
+      <version>5.8.6</version>
```

Comment:
really sorry for missing this at first round.
I think we write versions in properties, like ${rocksdbjni.version}